src/home/Home.jsx

```diff
@@ -1,8 +1,10 @@
 import React from "react";
 import './Home.css';
+import GetInTouch from '../Getintouch/getintouch'
 export default function Home() {
 
     return (
+        <React.Fragment>
```

Comment:
Please add proper formatting to the jsx?

Reply:
I'm sorry I didn't get this, can you please elaborate?

Reply:
I mean add proper indentation